### ISSUE TITLE
the loss

### ISSUE THREAD

**Issue by woqiaow** [zhaozhengChen/ReCAM#11]
Hi, Thanks for your code.
I want to ask why the loss value in my experiment is nun. Is this normal?
![image](https://user-images.githubusercontent.com/106650655/192426078-cf06fd62-5622-4d93-8757-3fc235c69178.png)

**Comment by zhaozhengChen** [comment]
No.
Maybe you can try to run it again, I haven't encountered this problem.

**Comment by woqiaow** [comment]
ok, thank you.

**Comment by YangjiaqiDig** [comment]
> Hi, Thanks for your code. I want to ask why the loss value in my experiment is nun. Is this normal? ![image](https://user-images.githubusercontent.com/106650655/192426078-cf06fd62-5622-4d93-8757-3fc235c69178.png)

did you solve it? I trained on my own dataset, also had this problem. However, after I decrease the cam_learning_rate it seems working (0.1 -> 0.01). Just curious if the learning rate is the only reason which leads to this Nan loss (gradient vanishing)

**Comment by woqiaow** [comment]
> > Hi, Thanks for your code. I want to ask why the loss value in my experiment is nun. Is this normal? ![image](https://user-images.githubusercontent.com/106650655/192426078-cf06fd62-5622-4d93-8757-3fc235c69178.png)
> 
> did you solve it? I trained on my own dataset, also had this problem. However, after I decrease the cam_learning_rate it seems working (0.1 -> 0.01). Just curious if the learning rate is the only reason which leads to this Nan loss (gradient vanishing)

I solved this problem. I set the batchsize back to 16 and it can run normally.

**Comment by YangjiaqiDig** [comment]
> > > Hi, Thanks for your code. I want to ask why the loss value in my experiment is nun. Is this normal? ![image](https://user-images.githubusercontent.com/106650655/192426078-cf06fd62-5622-4d93-8757-3fc235c69178.png)
> > 
> > 
> > did you solve it? I trained on my own dataset, also had this problem. However, after I decrease the cam_learning_rate it seems working (0.1 -> 0.01). Just curious if the learning rate is the only reason which leads to this Nan loss (gradient vanishing)
> 
> I solved this problem. I set the batchsize back to 16 and it can run normally.

Thank you, I will give it a try. actually, after decrease the LR, it can run normally sometimes but could fail as well
